src/llmcompressor/transformers/finetune/text_generation.py

```diff
@@ -65,27 +70,31 @@ def train(**kwargs):
     """
     CLI entrypoint for running training
     """
-    model_args, data_args, training_args = parse_args(**kwargs)
+    model_args, data_args, recipe_args, training_args, _ = parse_args(**kwargs)
```

Comment:
Don't we need the `output_dir` argument for training?

Reply:
its inside training_args.